src/GraphQL.SystemTextJson/ApolloTraceJsonConverter.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using GraphQL.Instrumentation;
+
+namespace GraphQL.SystemTextJson
+{
+    /// <summary>
+    /// Converts an instance of <see cref="ApolloTrace"/> to/from JSON.
+    /// </summary>
+    public class ApolloTraceJsonConverter : JsonConverter<ApolloTrace>
```

Comment:
Why do this class need? I thought that ApolloTrace serialization just works.

Reply:
This happened because of `NamingPolicy` removal?

Reply:
In a manner of speaking, the fact this did not exist was a bug.  If you look in the contract resolver for Newtonsoft.Json there exists code to detect properties of ApolloTrace and force a camelCase naming policy.  That code has never existed in this project for System.Text.Json.  So when using any other naming policy besides camelCase, serialization would be incorrect.  And the System.Text.Json serializer has always had a default of 'none' for the naming policy.  (At least since v4 for sure.)

Now the weird thing is that we had a test to ensure that ApolloTrace is serialized correctly.  But we had overridden the naming policy in the configuration of the test writer, so it was not testing a default scenario.  Moreover, it did not test what would happen when a custom naming policy was applied.  So it really wasn't testing what it was designed to test.  (Most likely the test was written for Newtonsoft.Json before we changed the tests to prefer System.Text.Json in 3.x)

Now the tests check both GraphQL.SystemTextJson and GraphQL.NewtonsoftJson.

> This happened because of `NamingPolicy` removal?

Yes -- in the tests project.  Not because the naming policy changed in the GraphQL.NewtonsoftJson project.

Reply:
> This happened because of `NamingPolicy` removal?

But also 'yes', since the default naming policy was set to camelcase in the converter, and even though it wasn't applying the camelCase conversion to `ExecutionResult` nodes, it was applying it to `extension` children.

So that's a breaking change if you consider that now extension dictionary entries will not have camelcase applied to them.  But I feel that to leave the default as camelCase is worse for two reasons:
1. It would not be unified behavior with the Newtonsoft converter
2. `ExecutionResult` nodes will now be converted to camelCase, even if another `NameConverter` was applied to the schema.

Reply:
Also, “none” naming strategy will need to be default for 5.x out of necessity because we cannot control the default serializer settings prior to the configuration delegates executing. Further, we cannot change after because there would be no way to distinguish a users desire for null versus the setting being unchanged from null.

Reply:
OK